packages/affix/index.ts

```diff
@@ -120,3 +120,10 @@ declare global {
     'w-affix': WarpAffix;
   }
 }
+declare module 'react' {
```

Comment:
Shouldn't this belong in the `react.ts` files as they are only to be used by people using react and not the web-component itself?

Reply:
I've moved all of this to a root types file now to get intellisense to play nice.
We need it to support both React with wrappers as well as React 19 without wrappers.